CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ For information on changes in released versions of Teku, see the [releases page]
 ### Additions and Improvements
 - Newly created databases will now use LevelDB for storage instead of RocksDB. This uses less memory and has proven to be more stable. Existing databases are unaffected and will continue to use RocksDB.
 - Support for automatic fail-over of eth1-endpoints.  Multiple endpoints can be specified with the new `--eth1-endpoints` CLI option. Thanks to Enrico Del Fante.
+- implement POST `/eth/v1/beacon/pool/sync_committees` to allow validators to submit sync committee signatures to the beacon node.
```

Comment:
Given the functionality that actually makes this work doesn't exist yet it's probably too early to add to the changelog.

Reply:
i did add the functionality, just not sure if we call it working if we're not in altair?

Reply:
Oh right - I was thinking of the bit of code that still has a `// FIXME`. I think it's fine to add.  I'll probably wind up summarising them as just a single "added endpoints in preparation for Altair" kind of thing in the final release notes but no harm in having the full details here so we can decide later.